Content.Server/Damage/Components/DamageOnLandComponent.cs

```diff
@@ -1,44 +1,28 @@
 using Content.Shared.Damage;
-using Content.Shared.Damage.Components;
-using Content.Shared.Throwing;
 using Robust.Shared.GameObjects;
 using Robust.Shared.Serialization.Manager.Attributes;
-using Robust.Shared.Prototypes;
-using Robust.Shared.IoC;
 using Robust.Shared.ViewVariables;
 
 namespace Content.Server.Damage.Components
 {
     [RegisterComponent]
-    public class DamageOnLandComponent : Component, ILand
+    public sealed class DamageOnLandComponent : Component
     {
         public override string Name => "DamageOnLand";
 
         [DataField("amount")]
         [ViewVariables(VVAccess.ReadWrite)]
-        private int _amount = 1;
+        public int Amount = 1;
 
         [DataField("ignoreResistances")]
         [ViewVariables(VVAccess.ReadWrite)]
-        private bool _ignoreResistances;
+        public bool IgnoreResistances;
 
         // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
         // Also remove Initialize override, if no longer needed.
-        [DataField("damageType")]
-        private readonly string _damageTypeID = "Blunt";
+        [DataField("damageType")] public readonly string DamageTypeId = "Blunt";
+
         [ViewVariables(VVAccess.ReadWrite)]
         public DamageTypePrototype DamageType = default!;
```

Comment:
ngl this field is just a weird way of caching the damagetypeprototype. change it if you want, i wont rip your head off if you dont tho

Reply:
I think it's how most of the damage stuff does it atm so I'll let whoever bulk changes them deal with it because I'm lazy :HECK: